### ISSUE TITLE
Stack Trace error, when starting TQVault.exe

### ISSUE THREAD

**Issue by ryksa92** [EtienneLamoureux/TQVaultAE#101]
For weeks TQVaultAE worked just perfectly, but suddenly, after exiting from game and trying to open it, a **Stack Trace** error popped up. 
- the version I'm using is **2.6.0**, the latest.
- the Vault actually starts, but incorrectly and all the vault options/character selection pages are blank by default, so I have to select them myself in each start.
- Game path auto-location feature turned off + set TQPath & TQITPath manually in config file.
- Also: **_Shared Stash is not visible anymore in Vault_**. I do not know what caused this. I'm using Enhanced Gameplay (it expands storage space ingame), but I have never any problems with that, also no update comes from the mod, so Vault suddenly faults.
- Here is the debug log file:
[tqdebug.txt](https://github.com/EtienneLamoureux/TQVaultAE/files/2663538/tqdebug.txt)

I hope I will not lose any of my precious items. Please help me. Thank you for your hard work and attention.
![tqvault_stacktraceerror](https://user-images.githubusercontent.com/41773535/49701737-4bf6fa00-fbf0-11e8-8efc-d46b19b66337.png)

**Comment by EtienneLamoureux** [comment]
Thanks for the detailed bug report, we'll try to look into this as soon as time permits. In the meantime, you can always try to recover a version of your vault from the [backups](https://github.com/EtienneLamoureux/TQVaultAE#features) the program generates to recover your items.

As a note, I'll add that we do not support any mods outside of the base game and its expansions, as changes in third-party mods are outside of our control.

**Comment by EtienneLamoureux** [comment]
Can you reproduce with the fully patched game and the latest release? Please make sure to open the faulty character in the game beforehand.

**Comment by TianheTengawa** [comment]
Update, I just got the 2.7 update and when I started the game, I got that message.

![Windows Forms Error](https://user-images.githubusercontent.com/51996874/59761980-1c424200-9264-11e9-9799-dbb9e73bb800.jpg)

When I do ignore, I don't see the Relic tab anymore in the vault. Maybe there is a relic in the relic vault from Ragnarok (or a recipe) that does not register correctly. I'll try to investigate. Right now I'm in the Darklands on Normal in the Ragnarok extension.
